### PR TITLE
homework_28_1

### PR DESCRIPTION
**Pull request by Alkey**

### CODE REVIEW

File: src/main/java/com/internet/shop/service/impl/AuthenticationServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package com.internet.shop.service.impl;
```
Comment (matviichuk92) [inline src/main/java/com/internet/shop/service/impl/AuthenticationServiceImpl.java]: можливо, для AuthenticationService треба створити окрему папку security

**Review by matviichuk92 — APPROVED** [review]
good job!

File: src/main/java/com/internet/shop/controllers/LoginController.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String login = req.getParameter("login");
+        String password = req.getParameter("password");
+        try {
+            User user = authenticationService.login(login, password);
+            req.getSession().setAttribute("user_id", user.getId());
```
Comment (KseniiaMakarova) [inline src/main/java/com/internet/shop/controllers/LoginController.java]: let's make `user_id` a constant

File: src/main/java/com/internet/shop/service/impl/AuthenticationServiceImpl.java (hunk 1 above)
Comment (KseniiaMakarova) [reply src/main/java/com/internet/shop/service/impl/AuthenticationServiceImpl.java]: I agree - let's put AuthenticationService and its impl in `security` package

File: src/main/java/com/internet/shop/service/impl/AuthenticationServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+import com.internet.shop.service.UserService;
+
+@Service
+public class AuthenticationServiceImpl implements AuthenticationService {
+    @Inject
+    private UserService userService;
+
+    @Override
+    public User login(String login, String password) throws AuthenticationException {
+        User user = userService.getByLogin(login).orElseThrow(() ->
```
Comment (KseniiaMakarova) [inline src/main/java/com/internet/shop/service/impl/AuthenticationServiceImpl.java]: I can't see changes in UserService and UserDao classes in this PR

File: src/main/web/WEB-INF/views/login.jsp [hunk 4]
```diff
@@ -0,0 +1,18 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Title</title>
```
Comment (KseniiaMakarova) [inline src/main/web/WEB-INF/views/login.jsp]: let's add the informative title

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/main/java/com/internet/shop/service/impl/AuthenticationServiceImpl.java (hunk 3 above)
Comment (Alkey) [reply src/main/java/com/internet/shop/service/impl/AuthenticationServiceImpl.java]: I implemented this method when I did HW 26.1

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**